ansible_galaxy/build.py

```diff
@@ -16,7 +16,7 @@
 
 log = logging.getLogger(__name__)
 
-ARCHIVE_FILENAME_TEMPLATE = 'v{version}.{extension}'
+ARCHIVE_FILENAME_TEMPLATE = '{name}-{version}.{extension}'
```

Comment:
Is the galaxy artifact import code ready for 'foo-1.0.0.tar.gz' style artifacts?  The 'v1.0.0.tar.gz' format existed only to match the github autocreated artifacts as best I can remember

The '{name}-{version}.{extension}' format is much much better than 'v{version}.{extension}' but wanted to mention server side stuff might need a matching tweak.

Reply:
I changed this according to Alex's spec for the server side upload. We can tweak it again, if needed.